pkg/clustermanager/cluster_manager.go

```diff
@@ -930,6 +933,37 @@ func (c *ClusterManager) ApplyBundles(ctx context.Context, clusterSpec *cluster.
 }
 
 func (c *ClusterManager) PauseEKSAControllerReconcile(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec, provider providers.Provider) error {
+	if clusterSpec.Cluster.IsSelfManaged() {
+		return c.pauseEksaReconcileForManagementAndWorkloadClusters(ctx, cluster, clusterSpec, provider)
+	}
+
+	return c.pauseReconcileForCluster(ctx, cluster, clusterSpec, provider)
+}
+
+func (c *ClusterManager) pauseEksaReconcileForManagementAndWorkloadClusters(ctx context.Context, managementCluster *types.Cluster, clusterSpec *cluster.Spec, provider providers.Provider) error {
+	clusters := &v1alpha1.ClusterList{}
+	err := c.clusterClient.GetObjects(ctx, fmt.Sprintf("clusters.%s", v1alpha1.GroupVersion.Group), clusterSpec.Cluster.Namespace, managementCluster.KubeconfigFile, clusters)
+	if err != nil {
+		return err
+	}
+
+	for _, w := range clusters.Items {
+		if w.ManagedBy() != clusterSpec.Cluster.GetName() {
+			continue
+		}
+		wSpec, err := cluster.BuildSpec(ctx, c.clusterClient.KubeconfigClient(managementCluster.KubeconfigFile), &w)
```

Comment:
do we need to build the whole spec here? it seems a little bit too much just to pause the cluster reconciliation